cleanlab/classification.py

```diff
@@ -610,7 +610,17 @@ def predict(self, *args, **kwargs) -> np.ndarray:
           Vector of class predictions for the test examples.
         """
         if self.default_clf:
-            X = force_two_dimensions(args[0])
+            if args:
+                X = args[0]
+            elif "X" in kwargs:
+                X = kwargs["X"]
+                del kwargs["X"]
+            elif "x" in kwargs:
+                X = kwargs["x"]
+                del kwargs["x"]
+            else:
+                raise ValueError("No input provided to predict")
```

Comment:
I think we can stick with supporting the just same argname as sklearn ([example](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html#sklearn.linear_model.LogisticRegression.predict)), so just checking for `X` should be fine, with a more descriptive error message otherwise.

Reply:
```suggestion
            else:
                raise ValueError("No input provided to predict, please provide X.")
```